File: .net Framework/BenefitPlanCompare/BenefitPlanCompare/Models/CopayDistribution.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BenefitPlanCompare.Models
{
    public class CopayDistribution : CompareEntity
    {
        public string AtlasRecordId { get; set; }
        public long? atlasBenefitId { get; set; }
        public long? atlasCoveragePhaseId { get; set; }
        public int? formularyTierId { get; set; }
        public decimal? memberResponsibilityAmt { get; set; }
        public decimal? memberResponsibilityPct { get; set; }
        public decimal? mfrResponsibilityPct { get; set; }
        public decimal? planDeductAmt { get; set; }
        public decimal? licsSubsidyPct { get; set; }
        public decimal? maxCoverageMonthly { get; set; }
        public decimal? maxCoverageYearl { get; set; }
        public decimal? maxOOPMonthly { get; set; }
        public decimal? maxOOPYearly { get; set; }
    }
}

File: UIO/Framework/AtlasUI/packages/local/plan/src/model/CopayDistribution.js
/**
 * Created by b2352 on 12/19/2016.
 */
Ext.define('Atlas.plan.model.CopayDistribution',{
    extend: 'Atlas.common.model.Base', //change to base when layer7 URL is ready
    fields: [
        {name: 'systemId',  type: 'number'},
        {name: 'coveragePhaseID',  type: 'number'},
        {name: 'coveragePhaseName',  type: 'string'},
        {name: 'formularyTierId',  type: 'number'},
        {name: 'formularyTierName',  type: 'string'},
        {name: 'memberResponsibilityPct',  type: 'number'},
        {name: 'memberResponsibilityAmt',  type: 'number'},
        {name: 'mfrResponsibilityPct',  type: 'number'},
        {name: 'pharmNetworkId',  type: 'number'},
        {name: 'licsSubsidyPct',  type: 'number'},
        {name: 'planDeductAmt',  type: 'number'},
        {name: 'maxOOPMonthly',  type: 'number'},
        {name: 'maxOOPYearly',  type: 'number'},
        {name: 'maxCoverageMonthly',  type: 'number'},
        {name: 'maxCoverageYearly',  type: 'number'}
    ],
    proxy: {
        url: 'plan/{0}/plancopaydistribution'
    }

});
